### DISCUSSION TITLE
pending payouts are misleading

### DISCUSSION THREAD

**Discussion by botnumberseven** [othub-io/othub-react#42] [General]
https://www.othub.io/staking/settings

Below 1651 are total estimated earnings, accumulated since the node was started. Out of them 1347 has been paid out already.
The original name was Estimated earnings, which was kind of aligned with the definition above.
But Pending looks misleading, I'd read it as I can expect 1651 more TRAC to be paid to me based on what my node booked by now.
But in reality i can expect only 1651 - 1347 = 304, that's what is Pending.

![image](https://github.com/othub-io/othub-react/assets/42500494/8c9b7db4-bb7c-4236-a0ad-fde9be39faea)

**Comment by CosmiCloud** [comment]
Fixed with commit be5c64a
